packages/experimental/RadioGroup/src/RadioGroup/RadioGroup.types.ts

```diff
@@ -31,6 +31,12 @@ export interface RadioGroupState extends RadioGroupProps {
    * Array of radio keys in the group
    */
   values?: string[];
+
+  /**
+   * Array of enabled radio keys in the group
+   * Windows-Specific Prop.
```

Comment:
Nit, the convention is more like `@platform win32`

Reply:
Replaced, thanks!